docs/faq/README.md

```diff
@@ -0,0 +1,86 @@
+# FAQ
+
+## Q: How to use private image repositories in OpenFunction?
+
+A: OpenFunction uses Shipwright (essentially using Tekton to integrate with Cloud Native Buildpacks) in the build phase to package the user function to the application image.
```

Comment:
(essentially using Tekton to integrate with Cloud Native Buildpacks)
=>
(which utilizes Tekton to integrate with Cloud Native Buildpacks)